connect/runtime/src/main/java/org/apache/kafka/connect/runtime/Worker.java

```diff
@@ -247,6 +248,19 @@ public boolean isSinkConnector(String connName) {
         }
     }
 
+    /*
+     * Retrieves the connector type by connector name
+     *
+     * @return the ConnectorType for the given connector name.
+     *  null if the worker does not manage a connector with the given name.
+     */
+    public ConnectorType getConnectorType(String connName) {
```

Comment:
nit: I know there are other cases in this class where a `get` prefix is used, but they predate the code being in Kafka and are mostly just fallout of IDE automation. for new stuff, we'd like to stick to kafka's style of not including `get`/`set` prefixes